MapboxNavigation/NavigationMapView.swift

```diff
@@ -46,6 +46,14 @@ open class NavigationMapView: MGLMapView, UIGestureRecognizerDelegate {
      */
     public var tapGestureDistanceThreshold: CGFloat = 50
     
+    /**
+     A collection of street road classes for which a congestion level substitution should occur.
+     
+     For any street road class included in the `trafficOverrideStreetsRoadClasses`, all route segments with an `CongestionLevel.unknown` traffic congestion level and a matching `MapboxDirections.MapboxStreetsRoadClass`
+     will be replaced with the `CongestionLevel.low` congestion level.
+     */
+    public var trafficOverrideStreetsRoadClasses: Set<MapboxStreetsRoadClass>? = nil
```

Comment:
This is a difficult concept to describe succinctly in a property name. The current name doesn’t sound like a property, but it doesn’t quite sound like a method either. Based on the documentation comment, `roadClassesWithOverriddenCongestionLevels` would be the most self-explanatory name I could think of.